### PR TITLE
[2197] Cleanup `cider-profile.el`

### PR DESCRIPTION
**Pull request by xiongtx**
Clean up code a bit and create prefix map.

cc @jmayaalv, @bbatsov

### CODE REVIEW

File: cider-profile.el [hunk 1]
```diff
@@ -24,10 +24,21 @@
 ;;; Code:
 
 (require 'cider-client)
-(require 'cider-mode)
 
 (defconst cider-profile-buffer "*cider-profile*")
 
+(defvar cider-profile-map
+  (let ((map (define-prefix-command 'cider-profile-map)))
```
Comment (bbatsov) [inline cider-profile.el]: You should also add the menu for the keymap here.

**Comment by bbatsov** [comment]
I'd unify the messages here with the style of messages in the tracing code, as they are inconsistent right now which is a bit odd.

**Comment by xiongtx** [comment]
I'm going to put this on hold until #2197 is clojure-emacs/cider-nrepl#502's [discussion](https://github.com/clojure-emacs/cider-nrepl/pull/502#issuecomment-363998577) is resolved.

**Comment by xiongtx** [comment]
> I'd unify the messages here with the style of messages in the tracing code

What style of messages? Can you elaborate?

**Comment by bbatsov** [comment]
I was mostly referring to this:

```
(defun cider--toggle-trace-var (sym)
  "Toggle var tracing for SYM."
  (let* ((trace-response (cider-sync-request:toggle-trace-var sym))
         (var-name (nrepl-dict-get trace-response "var-name"))
         (var-status (nrepl-dict-get trace-response "var-status")))
    (pcase var-status
      ("not-found" (error "Var %s not found" (cider-propertize sym 'fn)))
      ("not-traceable" (error "Var %s can't be traced because it's not bound to a function" (cider-propertize var-name 'fn)))
      (_ (message "Var %s %s" (cider-propertize var-name 'fn) var-status)))))
```

Capitalized messages with propertized var names.

**Comment by xiongtx** [comment]
👌. I'd have fixed it, but you already merged 🤷‍♂️.

**Comment by bbatsov** [comment]
I had some memory I already wrote this in the past, so I just assumed you didn't have time for it now, and it was fine to do it down the road. Then I saw my previous comment wasn't really there. 😆

**Pull request merged**